test-common/eos-test-utils.c

```diff
@@ -28,8 +28,10 @@
 #include <ostree.h>
 #include <string.h>
 
-#ifndef GPG_BINARY
-#error "GPG_BINARY is not defined"
+#include <libeos-updater-util/util.h>
+
+#ifndef GPG
```

Comment:
Changing `GPG_BINARY` to `GPG` in the test code is good, but could be done as a separate commit. For example: one commit to move the code (unchanged) to `test-common`; one commit to add the `additional_{directories,files,metadata}` stuff; and one commit to simply rename `{GPG,OSTREE,etc.}_BINARY` to `{GPG,OSTREE,etc.}` in the code and `Makefile.am`.

That way, the reviewer has to keep track of fewer changes in each commit, and so can approve those changes commit-by-commit at a more granular level.

Reply:
Ah whoops, I thought something may have been amiss - I'll do it in a separate commit.